service/ui.go

```diff
@@ -264,10 +264,10 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 	app.NoVersions = true
 
 	// User login and logout
-	app.AddRoute("/login").Handler(uis.loginPage).Get()
-	app.AddRoute("/login").Handler(uis.login).Post()
+	app.AddRoute("/login").Wrap(allowsCORS).Handler(uis.loginPage).Get()
```

Comment:
I'm unsure if GET login is called from Spruce. I checked locally and I didn't see it being requested.

Reply:
it's definitely not locally, but in staging and prod the `spruce.com/login` makes a GET request the login endpoint. I added it just to be on the safe side to avoid unexpected outcomes in staging. All of this will shortly be deleted anyways after John configures authentication for spruce.

Reply:
This part won't actually be deleted - what I'm doing should hopefully remove the cross-origin cookie hack. If spruce is calling this somehow we should fix it, because this returns an HTML page and certainly isn't useful